lib/crypto/base36.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var crypto = require('crypto')
```

Comment:
As this is a new module, shall we got all es6y on it (well the limited subset of es6 we can use in node 4, anyway)? I say break out the consts and lets!